### PR TITLE
tldr: add cache control and search example

### PR DESCRIPTION
**Pull request by KsRyY**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by pxgamer** [comment]
I was thinking about this, and wasn't sure whether this page should just follow the [Client Spec](https://github.com/tldr-pages/tldr/blob/dec9fb400bb7bac4778a752567f58be4815eb668/CLIENT-SPECIFICATION.md#command-line-interface). 🤔 Or if it should just match the [`tldr-node`](https://github.com/tldr-pages/tldr-node-client) client. I assume additional options were previously excluded as they may not exist in all clients?

**Comment by andrik** [comment]
Oh, I didn't think about that @pxgamer. You are right, but I would maintain `-u` example, even though it is optional in the spec

**Comment by KsRyY** [comment]
So I might add an notice that not every client can do that.

File: pages/common/tldr.md [hunk 1]
```diff
@@ -14,3 +14,15 @@
 - Get help for a git subcommand:
 
 `tldr {{git checkout}}`
+
+- Update the local cache and rebuild index:
+
+`tldr -u`
+
+- Clear the local cache:
```
Comment (sbrl) [inline pages/common/tldr.md]: I'm not sure about adding these last 2 examples, as they are not in the [official client specification](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md).

Comment (KsRyY) [reply pages/common/tldr.md]: So I would remove it. Applying.

Comment (KsRyY) [reply pages/common/tldr.md]: Changes committed to `patch-5`.

**Comment by KsRyY** [comment]
@sbrl @pxgamer removed these cache control examples. Thank you for pointing our my error.

**Comment by mebeim** [comment]
As @agnivade said, this page should contain only options that are listed as mandatory in the client specification, so that it can be compatible with all conforming clients. Appreciate the PR @KsRyY, but I think it should be closed unfortunately.

**Pull request closed without merging**